tests/composer.bats

```diff
@@ -0,0 +1,114 @@
+setup() {
+    cd /work/tests/composer
+}
+
+teardown() {
+    cd /work/tests/composer
+    if [ -d vendor ]; then rm -r vendor; fi
+    if [ -f composer.lock ]; then rm composer.lock; fi
+    cd /work/tests/composer_lock
+    if [ -d vendor ]; then rm -r vendor; fi
```

Comment:
Is all of this still needed? I thought you install the packages to the system place